src/main/java/com/vonage/quickstart/voice/AsrInput.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright  2020 Vonage
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.vonage.quickstart.voice;
+
+import com.vonage.client.incoming.InputEvent;
+import com.vonage.client.voice.ncco.InputAction;
+import com.vonage.client.voice.ncco.Ncco;
+import com.vonage.client.voice.ncco.SpeechSettings;
+import com.vonage.client.voice.ncco.TalkAction;
+import spark.Route;
+import spark.Spark;
+
+import java.util.Collections;
+
+public class AsrInput {
+
+    public static void main(String[] args) {
+        /*
+         * Route to answer incoming calls.
+         */
+        Route answerCallRoute = (req, res) -> {
+            TalkAction intro = TalkAction
+                    .builder("Please say something")
+                    .build();
+
+            SpeechSettings speechSettings = new SpeechSettings();
+            speechSettings.setEndOnSilence(1);
```

Comment:
Can we use ~3 seconds instead of 1 - when I run the code snippet, if I wait for the voice to stop talking it sends a completely silent event webhook.

Reply:
The [code snippet spec ](https://github.com/Nexmo/code-snippet-specs/blob/master/voice/handle-user-input-with-asr.md) says 1sec. We can change it to 3secs but then we would need to look into changing it in the spec and have others align with it.